Sources/XCTest/Public/Asynchronous/XCTWaiter.swift

```diff
@@ -117,7 +117,7 @@ open class XCTWaiter {
     internal var waitSourceLocation: SourceLocation?
     private weak var manager: WaiterManager<XCTWaiter>?
     private var runLoop: RunLoop?
-
+    private var runLoopSource: RunLoop._Source?
```

Comment:
I notice the `RunLoop._Source` type and several of its related RunLoop methods are marked as deprecated in Foundation… does that mean this PR will introduce new build warnings to the XCTest codebase? I wonder what we can do about that, since Swift has no way to "silence" those warnings.